.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1795,8 +1795,5 @@ jobs:
           method: kubeconfig
           kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
 
-      - name: Apply ingress changes
-        run: |
-          kubectl apply -f k8s/nginx/production/ingress.yaml
-          kubectl apply -f k8s/nginx/production/virtual-server.yaml
-          kubectl apply -f k8s/nginx/production/global-config.yaml
+      - name: Apply nginx changes
+        run: kubectl apply -f k8s/nginx/production/
```

Comment:
[update prod workflow to apply all files at once](https://github.com/airqo-platform/AirQo-api/pull/1526/commits/5f2b43ec20fdf3031c674c68ff2007d5a19618c6)